lib/orbit_web/controllers/auth_controller.ex

```diff
@@ -0,0 +1,127 @@
+defmodule OrbitWeb.AuthController do
+  use OrbitWeb, :controller
+  plug(Ueberauth)
+
+  alias OrbitWeb.Auth.Auth
+  require Logger
+
+  def login_page(conn, _params) do
+    if Auth.logged_in_user(conn) do
+      conn
+      |> redirect_login_successful()
```

Comment:
**nitpick:** we usually don't pipeline function calls if it results in just a single pipeline step.